File: benchmark/results/java-invocation-inline/run-004/b-baseline/output/com/metaengine/demo/identity/services/AuthService.java
package com.metaengine.demo.identity.services;

import java.util.List;
import com.metaengine.demo.identity.aggregates.User;

/** AuthService domain service. */
public class AuthService {
    /** Authenticate and create a User session. */
    public User create(User input) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Find a User by id; returns null if not found. */
    public User findById(String id) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** List up to limit Users. */
    public List<User> list(int limit) {
        throw new UnsupportedOperationException("not implemented");
    }

    /** Delete a User session by id. */
    public void delete(String id) {
        throw new UnsupportedOperationException("not implemented");
    }
}

File: benchmark/results/python-multilang/run-004/b-baseline/output/metaengine_demo/identity/services/auth_service.py
"""AuthService for the identity domain."""
from typing import Optional

from metaengine_demo.identity.aggregates.user import User


class AuthService:
    """Service handling user authentication."""

    def create(self, input: Optional[User]) -> User:
        """Authenticate and create session for a User."""
        raise NotImplementedError("not implemented")

    def findById(self, id: str) -> Optional[User]:
        """Find an authenticated User by id."""
        raise NotImplementedError("not implemented")

    def list(self, limit: float) -> list[User]:
        """List authenticated Users up to limit."""
        raise NotImplementedError("not implemented")

    def delete(self, id: str) -> None:
        """End the session of a User by id."""
        raise NotImplementedError("not implemented")
